Source/Public/Update-VSTeamGitRepositoryDefaultBranch.ps1

```diff
@@ -0,0 +1,61 @@
+# Sets the default branch for a Git repository in your Azure DevOps or Team Foundation Server account.
+#
+# Get-VSTeamOption 'git' 'repositories'
+# id              : 225f7195-f9c7-4d14-ab28-a83f7ff77e1f
+# area            : git
+# resourceName    : repositories
+# routeTemplate   : {project}/_apis/{area}/{resource}/{repositoryId}
+# http://bit.ly/Add-VSTeamGitRepository
+function Update-VSTeamGitRepositoryDefaultBranch {
+   [CmdletBinding()]
+   param(
+      [parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
+      [string] $Name,
+      [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
+      [vsteam_lib.ProjectValidateAttribute($false)]
+      [ArgumentCompleter([vsteam_lib.ProjectCompleter])]
+      [string] $ProjectName,
+      [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
+      [string] $DefaultBranch
+   )
+   begin {
+      try {
+         # $Repo = Get-VSTeamGitRepository -Name $Name -ProjectName $ProjectName
```

Comment:
Why are you not using this cmdlet? We are fine to use existing cmdlet to make more complex ones.

Reply:
Honestly, I believe I intended to be revert this once the tests were working, but this is related to the aforementioned issue with Pester.

Reply:
Fixed, re-committing.